package.json

```diff
@@ -78,7 +78,6 @@
     "eslint-import-test-order-redirect": "file:./tests/files/order-redirect",
     "eslint-module-utils": "file:./utils",
     "eslint-plugin-eslint-plugin": "^2.3.0",
-    "eslint-plugin-import": "2.x",
```

Comment:
In the initialization of the cli engine, it is necessary to register it in the dependency in order to read the config. 
The plugin added to the cli engine is used for linting.

Reply:
This behavior may be a bug in eslint, but it may not be fixed prior to v8.

Reply:
yeah, i've been playing with it locally; removing this dev dep is much more complex.